cmake-build-debug/CMakeFiles/3.10.2/CMakeCCompiler.cmake

```diff
@@ -0,0 +1,73 @@
+set(CMAKE_C_COMPILER "/usr/bin/cc")
```

Comment:
I think you've added an extra directory by mistake. I'll just force push myself.